languages/elixir/exercises/concept/conditionals/.docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+The Elixir `atom` type represents a fixed constant. An atom's value is simply its own name.  This gives us a type-safe way to interact with data.  Atoms can be defined as follows:
+
+```elixir
+# All atoms are preceeded with a ':' then follow with alphanumeric snake-cased characters
+variable = :atom
+```
+
+_Atoms_ are internally represented by an integer in a lookup table, which are set automatically.  It is not possible to change this internal value.
+
+Often, we want to write code that can branch based on a condition. While there are many ways to do this in Elixir, one of the simplest ways is using a `cond` function.
+
+At its simplest, a `cond` follows the first path that evaluates to _true_:
```

Comment:
Maybe to further make the introduction concise, maybe just have the second example? I feel that would be enough.